scipy/stats/tests/test_morestats.py

```diff
@@ -653,6 +653,33 @@ def test_alternatives(self):
         res = stats.binom_test(51, 235, p=1./6, alternative='two-sided')
         assert_almost_equal(res, 0.0437479701823997)
 
+    # Expected results here are from R 3.6.2 binom.test
```

Comment:
Tests look good. I checked the numbers [here](https://rdrr.io/snippets/).
```
binom.test(20, 100, 0.25, alternative = "two.sided")
```

Reply:
👍